zap/src/main/java/org/parosproxy/paros/security/SslCertificateServiceImpl.java

```diff
@@ -161,7 +163,7 @@ public KeyStore createCertForHost(CertData certData)
                         new X509CertificateHolder(caCert.getEncoded()).getSubject(),
                         BigInteger.valueOf(serial.getAndIncrement()),
                         new Date(System.currentTimeMillis() - Duration.ofDays(30).toMillis()),
-                        new Date(System.currentTimeMillis() + Duration.ofDays(1000).toMillis()),
+                        new Date(System.currentTimeMillis() + Duration.ofDays(825 - 30).toMillis()),
```

Comment:
I've entered it this way to that it's more clear what's going on (yes 795 might have been more simple but I'm not sure it would be as clear to whoever works on this code next).

Reply:
It's possible that the cert could have a longer validity than 825 when the values between the two `System.currentTimeMillis()` differ. The current time could be sourced once and referenced from a variable.

The root certificate currently expires after 365 days, so maybe the intermediate could have a much shorter validity too.

Reply:
I'll adjust the clock call, that makes sense.

The ultimate plan is for an options UI to be put in place for this at which point users will be able to control the validity period.